src/rpc/guards/HandshakeCompletedGuard.ts

```diff
@@ -41,10 +36,23 @@ export class HandshakeCompletedGuard extends AGuard<ARpcService<ARpcMethods>> {
         const peerAddress = profile?.evmAddress?.toString();
 
         this.service.logger.warn(
-            "Handshake never completed; aborting RPC execution",
+            "No peer profile for transport; disconnecting",
             {
-                peerAddress
+                peerAddress,
+                service: rpc.service,
+                method: rpc.method
             }
         );
+
+        // With the new connection semantics, any guarded RPC over an unverified
```

Comment:
avoid the word `new` in comments
new is only relevent during you code changes. a week from now `new` means nothing